transport_nantes/topicblog/views.py

```diff
@@ -305,26 +305,38 @@ def get_template_names(self):
 class SendableObjectMixin:
     """ Define the sending by email behaviour for TopicBlog Objects """
 
+    # Defines the concrete subclass of SendRecordBase that we use to
+    # record this sending operation.
     send_record_class = None
+    # Used for determining whether a message is being rendered for
+    # email or for web.
     context_appropriate_base_template = None
+    # Defines the concrete subclass of sendable that we are treating.
     base_model = None
 
     def get_last_published_email(self, tb_slug: str) -> base_model:
-        """Get the TB object with the most recent publication date
-        for a given slug.
+        """Fetch the currently published email object.
+
+        Fetch the base_model instance in this slug class with most
+        recent publication_date.
+
         """
         tb_objects = self.base_model.objects.filter(
             slug=tb_slug,
             # Not implemented yet, but a published TBObject should be
-            # the only one with a publication date. For now it picks the
-            # most recent.
+            # the only one with a publication date. For now it picks
+            # the most recent.  **BUG**: assumes precisely one object
+            # returned, no order_by.
             publication_date__isnull=False
         )
         if len(tb_objects) > 1:
```

Comment:
Given the order_by and the fact that now multiple items might have a publication date (at least we expect that behavior to happen), checking the length for anything other than 0 seems useless

Reply:
Right, but currently we signal an error if it's great than one.  So we'll change the comment again (and remove the "**BUG**") when we fix this.